src/dir.targets

```diff
@@ -172,4 +172,23 @@
 
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="One or more tests failed while running tests from '$(MSBuildProjectName)' please check log for details!" />
   </Target>
+
+  <!-- Generate coverage for individual projects. -->
+  <Target Name="GenerateIndividualCoverageReport"
+    AfterTargets="BuildAndTest"
+    Inputs="$(CoverageReportDir)\*.coverage.xml"
```

Comment:
These inputs / outputs are what you want. You should make them specific for this one project.